public/scripts/gameHandler.js

```diff
@@ -33,6 +43,44 @@ function displayRole(role) {
   description.textContent = role.description
 }
 
+function showIsActivePlayer(activePlayer) {
+  const playerListArray = Array.from(playerList.childNodes)
+  const players = playerListArray.filter(function(node) {
+    return node.nodeName === 'LI'
+  })
+  for (let player of players) {
+    if (player.innerText === activePlayer) {
+      player.innerText = '> ' + activePlayer
+    } else if (player.innerText.substring(0, 2) === '> ') {
+      player.innerText = player.innerText.substring(2)
+    }
+  }
+}
+
+function progressGame(newJob, roundCount, activePlayer) {
+  roundCounter.textContent = `ROUND ${roundCount}`
+  showIsActivePlayer(activePlayer)
+  if (newJob === 'chat') {
+    wordCounter.hidden = false
+    customInput.hidden = false
```

Comment:
this will likely require some changes when merging in with my builder stuff, but I'll deal with that later